keras/engine/topology.py

```diff
@@ -2348,6 +2346,41 @@ def load_weights(self, filepath):
             K.batch_set_value(weight_value_tuples)
         f.close()
 
+    def prepare_config(self):
```

Comment:
I find that having `prepare_config` and `get_config` could be confusing... Maybe better naming would help?